spec/lib/hyrax/controlled_vocabulary/importer/language_spec.rb

```diff
@@ -11,7 +11,8 @@
   end
 
   let(:instance) { described_class.new }
-  let(:rdf_path) { Gem.loaded_specs['hyrax'].full_gem_path + "/.internal_test_app/tmp/lexvo_2013-02-09.rdf" }
+  let(:rdf_path) { "#{root_path}/tmp/lexvo_2013-02-09.rdf" }
+  let(:root_path) { ENV.fetch('RAILS_ROOT', "#{Gem.loaded_specs['hyrax'].full_gem_path}/.internal_test_app") }
```

Comment:
why not Rails.root here?

Reply:
oh, that probably works. i'll give it a try.

Reply:
okay, i think i've got this now.